src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const styles = sourceString.split(';');
+
+  for (const style of styles) {
+    const newStyle = style.trim();
+
+    if (newStyle) {
+      const [key, value] = newStyle.split(':');
+
+      if (key && value) {
+        result[key.trim()] = value.trim();
+      }
+    }
+  }
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. For example, you can use `split`, `filter`, and `reduce` to achieve the same result in a more functional programming style.